pages/settings/account/index.tsx

```diff
@@ -298,6 +298,17 @@ const AccountSettings = () => {
       <div className={styles.container}>
         <div className={styles.container__child}>
           <h1>Settings</h1>
+          <QueryInfo
+            data={null}
+            loading={false}
+            error={error?.message || ''}
+            texts={{
+              loading: '',
+              data: '',
+              error:
+                'Oops, something went wrong when getting your info. Please try again'
```

Comment:
Should we also start organizing our error messages in a constant file at this point?
NIT: The error message would sound better like this 'We encountered an issue while retrieving your information. Please try again.'

Reply:
I agree. Good idea!